dashboards-observability/public/components/application_analytics/home.tsx

```diff
@@ -160,7 +161,7 @@ export const Home = (props: HomeProps) => {
     return http
       .post(`${CUSTOM_PANELS_API_PREFIX}/panels`, {
         body: JSON.stringify({
-          panelName: `${appName}'s Panel`,
+          panelName: `${appName}'s Dashboards`,
```

Comment:
if panel is singular should it be dashboard instead of dashboards?

Reply:
yes i think i'll need to use dashboard here